static/src/javascripts/projects/commercial/types.d.ts

```diff
@@ -24,3 +24,8 @@ type GuAdSize = {
 	height: number;
 	toString: (_: void) => string;
 };
+
+type RuleSpacing = {
+	minAbove: number;
+	minBelow: number;
+};
```

Comment:
Is this type specific enough to Spacefinder/Spacefiller to be located with the rest of the types in Spacefiller?

Reply:
Yes definitely, I think it was just left behind by my careless copy/pasting 😳